stories/Molecules/Blocks/Lightbox/Lightbox.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./lightbox.scss";
+export const Lightbox = ({}) => (
+  <>
```

Comment:
@harishkumar-srijan I don't think we need this fragment

Reply:
Done